src/utils.js

```diff
@@ -186,6 +187,14 @@ const snakeCaseFormData = (formData) => {
   return transformedData;
 };
 
+const snakeCaseObjectToForm = (payload) => {
+  // transforms an object to a snake_cased FormData object
+  snakeCaseObject(payload);
+  const formData = new FormData();
+  Object.keys(payload).forEach(key => formData.append(key, payload[key]));
```

Comment:
nit: `snakeCaseObject` mutates the object passed in, which isn't part of the explicit contract for `snakeCaseObjectToForm`.  I would at least make a note that `payload` will be altered, or perform the operations on a copy of the object to leave the original untouched. 
```suggestion
  const snakeCaseData = snakeCaseObject({...payload});
  const formData = new FormData();
  Object.keys(snakeCaseData).forEach(key => formData.append(key, payload[key]));
```